.gitignore

```diff
@@ -41,3 +41,7 @@ Testing/functional/e2e/debug.log
 
 # CSV files and Logs
 *.log
+
+Apps/WebClient/src/ClientApp/.env.local
```

Comment:
Nit: could be grouped with `*.local.json` under `# Local settings files` on line 24 and shortened to `.env.local`.